examples/chess.py

```diff
@@ -145,4 +145,7 @@ def quit(self):
     white.do_move (move_black)
     print('tail of loop')
 
-g.quit()
+try:
+    g.quit()
+except NameError:
```

Comment:
I think the linter identified a real problem with the example here - this fix is just sweeping it under the carpet.